src/app/pattern-language-management/pattern-language-management/pattern-language-management.component.ts

```diff
@@ -38,75 +37,75 @@ import { UriConverter } from '../../core/util/uri-converter';
 
 export class PatternLanguageManagementComponent implements OnInit {
 
-    patternLanguages: Array<PatternLanguageModel>;
+  patternLanguages: Array<PatternLanguageModel>;
 
-    constructor(
-        private cdr: ChangeDetectorRef,
-        private router: Router,
-        private activatedRoute: ActivatedRoute,
-        private zone: NgZone,
-        private dialog: MatDialog,
-        private _cookieService: CookieService,
-        private _toasterService: ToasterService,
-        private patternLanguageService: PatternLanguageService) {
-    }
+  constructor(
+    private cdr: ChangeDetectorRef,
+    private router: Router,
+    private activatedRoute: ActivatedRoute,
+    private zone: NgZone,
+    private dialog: MatDialog,
+    private _cookieService: CookieService,
+    private _toasterService: ToasterService,
+    private patternLanguageService: PatternLanguageService) {
+  }
 
-    // function used to sort the patternlanguages (by name)
-    private static sortPatternLanguages(pl1: PatternLanguageModel, pl2: PatternLanguageModel): number {
-      if (pl1.name > pl2.name) {
-        return 1;
-      }
-      if (pl1.name < pl2.name) {
-        return -1;
-      }
-      return 0;
+  // function used to sort the patternlanguages (by name)
+  private static sortPatternLanguages(pl1: PatternLanguageModel, pl2: PatternLanguageModel): number {
+    if (pl1.name > pl2.name) {
+      return 1;
     }
-
-    ngOnInit() {
-      this.patternLanguages = Array.from<PatternLanguageModel>(this.activatedRoute.snapshot.data.patternlanguages.values())
-        .sort(PatternLanguageManagementComponent.sortPatternLanguages);
+    if (pl1.name < pl2.name) {
+      return -1;
     }
+    return 0;
+  }
 
-    // reload the current data from https://purl.org/patternpedia that contains all patternlangauges
-    async reloadPatternRepo() {
-      this.patternLanguageService.getPatternLanguages()
-        .pipe(
-          map(result => result.sort(PatternLanguageManagementComponent.sortPatternLanguages)))
-        .subscribe(result => {
-          this.patternLanguages = result;
-          this._toasterService.pop('success', 'Reloaded Pattern Languages');
-          this.cdr.detectChanges();
-          return result;
-        });
-      this.cdr.detectChanges();
-    }
+  ngOnInit() {
+    this.patternLanguages = Array.from<PatternLanguageModel>(this.activatedRoute.snapshot.data.patternlanguages.values())
+      .sort(PatternLanguageManagementComponent.sortPatternLanguages);
+  }
 
-    navigateToPL(pl: UriEntity): void {
-      this.zone.run(() => {
-        this.router.navigate([UriConverter.doubleEncodeUri(pl.uri)], { relativeTo: this.activatedRoute });
+  // reload the current data from https://purl.org/patternpedia that contains all patternlangauges
```

Comment:
I think this comment is really really old since we switched to the database instead of this webside approx. 1 year ago? :D :D